examples/rest-api/build.sh

```
@@ -18,19 +18,31 @@ fi
 echo "✅ AWS SSO session active"
 
-# Export current AWS credentials
-echo "🔑 Exporting AWS credentials..."
-eval $(aws configure export-credentials --format env)
+# Create temporary AWS credentials file
+echo "🔑 Creating temporary AWS credentials file..."
+TEMP_CREDS_FILE=$(mktemp)
 
-# Build Docker image
+# Export current AWS credentials to temporary file
+eval $(aws configure export-credentials --format env)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)